### PR TITLE
[Merged by Bors] - feat(AlgebraicGeometry/EllipticCurve/DivisionPolynomial/Degree): add further degree lemmas

### PR DESCRIPTION
**Pull request by Multramate**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

File: Mathlib/AlgebraicGeometry/EllipticCurve/DivisionPolynomial/Degree.lean [hunk 1]
```diff
@@ -11,20 +11,32 @@ import Mathlib.Tactic.ComputeDegree
...
+
+## Mathematical background
+
+Let $W$ be a Weierstrass curve over a commutative ring $R$. By strong induction,
  * $\tilde{\Psi}_n = \tfrac{n}{2}X^{\tfrac{n^2 - 4}{2}} + \dots$ if $n$ is even,
  * $\tilde{\Psi}_n = nX^{\tfrac{n^2 - 1}{2}} + \dots$ if $n$ is odd,
  * $\Psi_n^{[2]} = n^2X^{n^2 - 1} + \dots$, and
  * $\Phi_n = X^{n^2} + \dots$.
 
+In particular, when $R$ is an integral domain of characteristic different from $n$, the univariate
```
Comment (alreadydone) [inline Mathlib/AlgebraicGeometry/EllipticCurve/DivisionPolynomial/Degree.lean:23]: this should say "not dividing n" right?

Comment (Multramate) [reply Mathlib/AlgebraicGeometry/EllipticCurve/DivisionPolynomial/Degree.lean:23]: Yes... this got merged a bit too quickly, but I'll make the change in a different PR.

**Pull request closed without merging**